docs/resources/device_subnet_routes.md

```diff
@@ -3,12 +3,18 @@
 page_title: "tailscale_device_subnet_routes Resource - terraform-provider-tailscale"
 subcategory: ""
 description: |-
-  The device_subnet_routes resource allows you to configure subnet routes for your Tailscale devices. See https://tailscale.com/kb/1019/subnets for more information.
+  "The device_subnet_routes resource allows you to configure enabled subnet routes for your Tailscale devices. See https://tailscale.com/kb/1019/subnets for more information.
+  Routes must be both advertised and enabled for a device to act as a subnet router or exit node. Advertised routes must be set directly on the device and cannot be managed through Terraform. If a device has advertised routes, they are not exposed to traffic until they are enabled. Conversely, if routes are enabled before they are advertised, they are not available for routing until the device in question has advertised them.
```

Comment:
Nit:

"advertised routes must be set directly on the device" -> "routes must be advertised directly from the device"

Reply:
Nit:

"If a device has advertised routes" -> "If a device is advertising routes"

Reply:
Nit:

"the device in question has advertised them" -> "the device in question is advertising them"